Cargo.toml

```diff
@@ -24,6 +24,7 @@ toml = "0.5"
 proc-macro2 = "1"
 quote = "1"
 heck = "0.3"
+convert_case = "0.4.0"
```

Comment:
We have `heck` already to do case conversions, let's not introduce another dependency just because.

Reply:
Got it. Agreed.